### ISSUE TITLE
Depricated 1.0 API

### ISSUE THREAD

**Issue by jasonjayr** [srchulo/DigitalOcean#8]
We're using this library to manage our DigitalOcean connection, and they've reached out to us because they noticed it's still using the 1.0 API:

>    You should have recently received a system message from DigitalOcean to remind you of the pending API v1 deprecation on November 9th.                                                                                                                                                                                  
>    I'm following up with you now as we've seen that you are still actively using API v1 for some of your deployments. We want to be sure                                                                                                                                                                                  
>    that you have ample time to put an alternative in place using API v2.          

Is there an update in the works, or would patches be welcome?

**Comment by srchulo** [comment]
Hi Jason,

So I have a version right now that works with version 2 of the API, however I have been unable to add documentation because I have just started a new job and become very busy. I'm going to try to release before November 9th and get the documentation done. I've contemplated releasing it without complete documentation, since version 1 won't work at all anymore, but I would prefer to release it with documentation. Let me know if you have any thoughts or I didn't answer any of your questions. 

Thanks,
Adam

**Comment by jasonjayr** [comment]
Feel free to git push it up to github, and I can help with the docs if need be prior to a full CPAN release. Our own internal systems are scantily documented, I'm used to it by now :)

**Comment by srchulo** [comment]
Thanks a lot, Jason. I really appreciate the help! It's on my personal computer, so I'll have to push it later tonight when I get home. I also haven't had a chance to write tests for this version (a lot of the old tests didn't work anymore since some of the API had to change), not saying that you also have to do that, but just so you're aware that the only testing that's really been done on the code was the manual testing that I did as I developed it. I'll shoot you an email later tonight once I push it out!

Adam
